docs/USAGE-GUIDE.md

```diff
@@ -485,7 +485,13 @@ Applications may want to know which certificate was used by a server for authent
 
 Additionally s2n-tls has functions for parsing certificate extensions on a certificate. Use `s2n_cert_get_x509_extension_value_length()` and `s2n_cert_get_x509_extension_value()` to obtain a specific DER encoded certificate extension from a certificate. `s2n_cert_get_utf8_string_from_extension_data_length()` and `s2n_cert_get_utf8_string_from_extension_data()` can be used to obtain a specific UTF8 string representation of a certificate extension instead. These functions will work for both RFC-defined certificate extensions and custom certificate extensions.
 
-### OCSP Stapling
+### Certificate Revocation
+
+There are some circumstances in which a CA might wish to revoke one of its issued certificates. For instance, if an issued certificate’s private key is suspected to be leaked, the certificate should no longer be trusted. And thus, the CA will want to inform those validating certificate of the leak via a method of certificate revocation.
+
+s2n-tls supports two methods of certificate revocation validation: OCSP stapling, and CRLs. In general, OCSP stapling should be used when possible, and CRLs should be used in situations where OCSP stapling isn't supported or otherwise can't be used. For instance, if a server wants to determine if a client certificate is revoked, the server can request that the client send a stapled OCSP response, but there is no way to enforce that an unknown client will actually send the response. The client, for example, may not support OCSP stapling. As such, CRLs could be used in this case instead.
```

Comment:
I think this can most easily be summarized as the difference between OCSP being optional, and CRLs being always available to check. I revised this section with this distinction. Let me know if it's still unclear!